CLK1 paper blog post (#102)
rokitalab/rokitalab.com: _posts/2026-03-10-clk1-paper-accepted.md

---
title: Our study of RNA splicing across pediatric brain tumors is now published in Neuro-Oncology Pediatrics!

> author: Jo Lynne Rokita

Reviewer: ```suggestion
author: Patricia Sullivan
```

tags:
  - publication

---

> We are excited to share the publication of our latest study, “Alternative splicing in pediatric central nervous system tumors highlights oncofetal candidate _CLK1_ exon 4,” now published in [Neuro-Oncology Pediatrics](https://doi.org/10.1093/neuped/wuag009). Led by co-first authors [Dr. Ammar Naqvi](https://d3b.center/team-members/naqvi/) and [Dr. Patricia Sullivan](https://rokitalab.com/members/06-patricia-sullivan.html), this work represents years of collaborative effort across institutions and builds on our ongoing mission to understand the molecular programs that shape pediatric brain tumors.

Reviewer: ```suggestion
We are excited to share the publication of our latest study, “Alternative splicing in pediatric central nervous system tumors highlights oncofetal candidate _CLK1_ exon 4,” now published in [Neuro-Oncology Pediatrics](https://doi.org/10.1093/neuped/wuag009). 
Led by co-first authors [Dr. Ammar Naqvi](https://d3b.center/team-members/naqvi/) and [Dr. Patricia Sullivan](https://rokitalab.com/members/06-patricia-sullivan.html), this work represents years of collaborative effort across institutions and builds on our ongoing mission to understand the molecular programs that shape pediatric brain tumors.
```
one sentence per line

> In this study, we analyzed RNA splicing across 729 pediatric CNS tumors and found that different tumor types use distinct splicing patterns. These patterns allowed us to identify groups of tumors defined by how their RNA is spliced. We also highlight a candidate oncofetal splicing event in _CLK1_, involving inclusion of exon 4, that may represent a potential therapeutic vulnerability. Importantly, exon-level regulation of _CLK1_ was linked to prognosis in certain aggressive tumors, suggesting that specific splicing events may influence disease behavior and treatment strategies beyond traditional factors such as age, molecular subtype, or extent of surgical resection.

Reviewer: ```suggestion
In this study, we analyzed RNA splicing across 729 pediatric CNS tumors and found that different tumor types use distinct splicing patterns, allowing us to identify groups of tumors defined by how their RNA is spliced. 
We highlight a candidate oncofetal splicing event in _CLK1_, involving inclusion of exon 4, that may represent a potential therapeutic vulnerability. 
Importantly, exon-level regulation of _CLK1_ was linked to prognosis in certain aggressive tumors, suggesting that specific splicing events may influence disease behavior and treatment strategies beyond traditional factors such as age, molecular subtype, or extent of surgical resection.
```

Congratulations to the outstanding team that made this study possible, and to the many collaborators spanning our time at Children’s Hospital of Philadelphia and now at Children’s National Hospital.

> Special thanks to the patients and families, the Children's Brain Tumor Network for creating this dataset, the anonymous donors to the Brain Tumor Institute, Gabriella Miller Kids First Data Resource Center for supporting us with cloud credits, and every member of the team whose dedication made this possible.

Reviewer: ```suggestion
Special thanks to the patients and families, the Children's Brain Tumor Network and Pediatric Neuro-Oncology Consortium for creating this dataset, the anonymous donors to Children's National Brain Tumor Institute, Gabriella Miller Kids First Data Resource Center for supporting us with cloud credits, and every member of the team whose dedication made this possible.
```
